BEIMA.Backend.Test/RequestFactory.cs

```diff
@@ -44,5 +50,23 @@ public static HttpRequest CreateHttpRequest(RequestMethod requestMethod, Diction
             }
             return reqMock.Object;
         }
+
+        public static HttpRequest CreateMultiPartHttpRequest(string data)
+        {
+            DefaultHttpContext httpContext = new DefaultHttpContext();
+
+            var formKeys = new Dictionary<string, StringValues>();
+            formKeys.Add("data", data);
+
+            var formFiles = new FormFileCollection();
+            var fileStream = new ByteArrayContent(Encoding.ASCII.GetBytes("TestOne")).ReadAsStream();
+            var photoStream = new ByteArrayContent(Encoding.ASCII.GetBytes("TestTwo")).ReadAsStream();
+            formFiles.Add(new FormFile(fileStream, 0, fileStream.Length, "files", "file.txt"));
+            formFiles.Add(new FormFile(photoStream, 0, photoStream.Length, "photos", "photo.txt"));
```

Comment:
I think we should use the method arguments to populate the files and photos as opposed to hard coding them for every request that we test.

Reply:
Moved those to a method argument